api/app/Http/Controllers/Api/Auth/AuthController.php

```diff
@@ -71,4 +79,11 @@ public function me(GetAuthenticatedUserAction $action, UserArrayPresenter $userA
 
         return $this->successResponse($userArrayPresenter->present($response->getUser()));
     }
+
+    public function refresh()
+    {
+        $response = $this->refreshTokenAction->execute();
+
+        return $this->successResponse($this->refreshTokenPresenter->present($response));
```

Comment:
There is no need to use a separate presenter. The endpoint must return the same data as the login endpoint.